include/dca/math/nfft/dnfft_1d.hpp

```diff
@@ -48,7 +48,9 @@ class Dnfft1D {
   using ThisType = Dnfft1D<ScalarType, WDmn, PDmn, oversampling, mode>;
   using ElementType = ScalarType;
 
+public:
```

Comment:
What is the reason of putting a second "public:" here?  I don't think it is necessary

Reply:
Just separating typedefs from methods. I suppose a space can achieve the same. Changed in 547604f176339765f2a44aac02708f7b0ec5dcb7